pkg/rpc/server/testdata/test_contract.go

```diff
@@ -74,10 +74,6 @@ func BalanceOf(addr []byte) int {
 	return amount
 }
 
-func Name() string {
-	return "Rubl"
-}
-
```

Comment:
Not a 83a3681611989e6679688098568cd557eb6848ea change.

Reply:
Fixed.